examples/ota-requestor-app/linux/BUILD.gn

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 Project CHIP Authors
```

Comment:
```suggestion
# Copyright (c) 2021 Project CHIP Authors
```

Reply:
Looks like most of the copyrights need to be updated; I won't go through and comment on them all but they should be audited

Reply:
Done. Keep in mind that `build_overrides` files are symlinks